service_management/azure/test/integration/blob/create_page_blob_test.rb

```diff
@@ -28,8 +28,8 @@
       subject.create_container container_name
     }
 
-    it 'creates a page blob' do
-      blob = subject.create_page_blob container_name, blob_name, length
+    it 'creates page blob with non uri encoded path' do
+      blob = subject.create_page_blob container_name, 'фбаф.jpg', length
       blob.name.must_equal blob_name
```

Comment:
@yaxia Yest it must be. Currently these tests aren't recorded and running in the travis-ci but i verified that this is wrong.